sdks/sdk_features_matrix.md

```diff
@@ -0,0 +1,21 @@
+= AeroGear SDK Feature Matrix
+
+
+== SDKs
+
+* Android (Native) - https://github.com/aerogear/aerogear-android-sdk
+* iOS (Native) - https://github.com/aerogear/aerogear-ios-sdk
+* Cordova (JS) - https://github.com/aerogear/aerogear-js-sdk
```

Comment:
What about React Native and Xamarin? Are they not still in?

Reply:
As per Wei's comment https://github.com/aerogear/proposals/pull/33#discussion_r176776523, I'll probably move this from proposals repo.
This is more of a 'state of the world' doc that what's commited to or on the roadmap.
Xamarin would likely be added and filled out in time as it's done